### DISCUSSION TITLE
Independent X or Y Axis Zooming

### DISCUSSION THREAD

**Discussion by sys732** [vega/vega-lite#9024] [Q&A]
I know that this has been asked before... 

Does anybody have a workaround to allow X or Y independent zoom

ie move the mouse pointer over one of the axis  zoom in/out using scroll wheel

very useful for analysis of  large Time series charts ie engineering, demographics ML training data

**Comment by eduardoraygoza-quidel** [comment]
You need two different params, one for `x` and one for `y`, use the `encodings` property respectively and the `zoom` property to define a condition.

[Example here](https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JAKyUAVhDYA7EABoQAEzjQATjRyZ289AHE2bMoLgByCAAIImNlADWlnJqhxLbUuss6EcSopAcJHUkBAg0AG1QWVC4NBAADwAtIwQAiDhTKEw0UEwATxw49BpZTDh1YiRBALhZKDZlUrJw1AjEkABdJQAvVPiAd3hMgEIIkbhSMsoIGBoAM0wAaTh8zpAAXyUmUuV46Gq1TYVo2Pj8lLY0pQysnNQ8wuKQUvLK6tr6xubW9vyu3r9dBDOCjCKTOqYGZzRYrNbHEA7WR7dAHUzhDbdFRITAoB4gBjqGroZQ4pCcWwOCCUKAQYgIzAhWQQeZsdRpNqgeY0QRveKkzAMBAzfIIJhsQQAXmlZn0AHk5fozJssch1PZ4sJZHElHUGk1ZGRch18dzMijseUAgUivFyggcGyPlsQP9TTRzfFnDRXNanvEAI4MJBlOg4mikTYbDZAA)
